### DISCUSSION TITLE
Distro Theming

### DISCUSSION THREAD

**Discussion by John2324** [SpiralLinux/SpiralLinux-project#64] [General]
I really like the approach of the distribution, with a focus on ease of installation. I was just wondering if there will be a theming option for this distro. If so, which DE would be the default for this distro?

**Comment by geckolinux** [comment]
Hi there, sorry I didn't understand the question. All the SpiralLinux editions support the normal GTK and Qt theming mechanisms and come with one theme preconfigured. And there is no default desktop environment, as SpiralLinux comes in 7 different live installable editions, each supporting a single desktop environment, which is what will also be used after installation.
